### ISSUE TITLE
Feature request: Single handed operation - Swipe right to go back

### ISSUE THREAD

**Issue by tomverbeure** [Dimillian/IceCubesApp#1378]
AFAIK, there are two ways to back:

* the Back button at the top left
* put your finger to the far left border of the screen and swiping right

Unfortunately, neither allow easy single handed operation. My thumb is not big enough to comfortably reach all the way left.

The default Swipe Right action is Reply. I would like the ability to select Back as an option.

Thanks!
Tom

**Comment by mfedoseev** [comment]
I concur. Using the app on an iPad, latest OS, can't go back from any view without reaching for that "< Back" button. It's a two-handed operation and hitting it on a 11" screen requires a decent aim. Would be great if instead of an action, there was a way to return to the timeline or a previous screen. Actions are great, but basic navigation is better!

**Comment by fridahjelm** [comment]
I agree, and I get this question a lot from new Mastodon users as well. It seems to be an expected and intuitive way for navigation to work.

**Comment by jcheroske** [comment]
Totally want this. The only swipe right motion my brain understands in this context is back.

**Comment by jamigibbs** [comment]
I came here to write a feature request for this so I'm glad to see that I'm not alone here. This would be a great UX improvement for me as well.

**Comment by RayJW** [comment]
Another suggestion that might work is adding the option to make the navigation bar swipeable. Apollo had that feature, and it makes one hand navigation much easier since the bottom nav bar is always visible and across the whole screen anyway, so swiping left to right on it to go back feels really intuitive.
